Microsoft.DependencyInjection.SolrNet.Tests/MicrosoftDependencyIntegrationFixture.cs

```diff
@@ -3,41 +3,39 @@
 using System;
 using Xunit;
 using SolrNet;
-using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.DependencyInjection;
+using Xunit.Abstractions;
 
 namespace Microsoft.DependencyInjection.SolrNet.Tests
 {
-    
     [Trait("Category","Integration")]
     public class MicrosoftDependencyIntegrationFixture {
-        private readonly IServiceProvider DefaultServiceProvider;
+        private readonly ITestOutputHelper testOutputHelper;
+        private readonly IServiceProvider defaultServiceProvider;
 
-        public MicrosoftDependencyIntegrationFixture()
+        public MicrosoftDependencyIntegrationFixture(ITestOutputHelper testOutputHelper)
```

Comment:
These are good refactors, but it would be best to have them in a seperate PR. Mixing refactors and functionality changes together into one PR makes the PR a little harder to review.

Reply:
I will move this change into a separate PR.

Reply:
Reverted.